pylint/utils/utils.py

```diff
@@ -328,6 +329,29 @@ def _check_csv(value: list[str] | tuple[str] | str) -> Sequence[str]:
     return _splitstrip(value)
 
 
+def _check_regexp_csv(value: list[str] | tuple[str] | str) -> Iterable[str]:
+    if isinstance(value, (list, tuple)):
+        yield from value
+    else:
+        # None is a sentinel value here
+        regexps: deque[deque[str] | None] = deque([None])
+        open_braces = False
+        for char in value:
+            if char == "{":
```

Comment:
Depending on the result of the discussion we should warn the user of deprecation here.

Reply:
wouldn't it be on line 346? That's where we split regular expressions.

Reply:
You're right, my bad.

Reply:
I know a year ago we were floating the idea of deprecations, but we're not ready to commit to a new plan IMO.